src/mapbox.common.ts

```diff
@@ -19,6 +19,16 @@ export interface LatLng {
   lng: number;
 }
 
+export interface Polygon {
```

Comment:
Don't necessarily need these since they aren't defining anything from MapBox, your call if you prefer it or not.